lib/src/components/card_rounded.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+/// It's a stateless widget that takes in a card and returns a card
```

Comment:
That's implementation details -- how about we describe __what__ the widget does, not __how__ it does it?

```suggestion
/// This is a [Card] with a rounded border, and an optional close button at the top right.
```